app/services/authentication/authenticator.rb

```diff
@@ -150,6 +150,8 @@ def default_user_fields
     end
 
     def update_user(user)
+      return user if user.suspended?
+
```

Comment:
This returns `user` because `user` is set in a higher level method:
https://github.com/forem/forem/blob/050729c729d4010ef2baabd1f1218d63ddaa910a/app/services/authentication/authenticator.rb#L49-L53